H/hidapi/build_tarballs.jl

```diff
@@ -19,10 +19,13 @@ mkdir build
 cd build
 
 if [[ ${target} == *-linux-* ]]; then
-    # We need this only to trick `configure` into thinking that udev is available,
-    # but we aren't going to actually use it
     apk add eudev-dev
-    linuxflags="-DHIDAPI_WITH_HIDRAW=false"
+    linuxflags="-DHIDAPI_WITH_LIBUSB=false"
+    export CPATH=/usr/include
```

Comment:
This looks wrong.

Reply:
Without the CPATH the `libudev.h` in `/usr/include` isn't found. But this is probably wrong because host build dependency (`libudev` from the package eudev-dev) would also need to be available for 32bit. Nowadays libudev is provided by systemd but I didn't find a jll for systemd...